webrtc.html

```diff
@@ -12936,6 +12936,11 @@ <h2>Confidentiality of Communications</h2>
       <p>A mechanism, <code><a>peerIdentity</a></code>, is provided that gives
       Javascript the option of requesting media that the same javascript cannot
       access, but can only be sent to certain other entities.</p>
+      <p>Communication certificates may be opaquely shared with
+      <code>postMessage</code> in anticipation of future needs. User agents are
```

Comment:
Nit: Shouldn't it be "using PostMessage"? I don't think we're sharing it with the message, we're using the message to share it.